new-client/src/utils/FetchWrapper.js

```diff
@@ -39,10 +39,12 @@ class FetchWrapper {
     this.options = {};
 
     // Hash is used for cacheBuster function.
-    //this.hash = process?.env?.REACT_APP_GIT_HASH || null;
-    this.hash = null;
-    //this.useCacheBuster = process?.env?.REACT_APP_USE_CACHE_BUSTER === "true" || false;
-    this.useCacheBuster = false;
+    // Lets get the values from generated meta-tags
+    this.hash = this.getMetaValue("hajk-client-git-hash") || "";
```

Comment:
Can we be sure that the meta tag will always be called `hajk-client-use-cache-buster`? I mean, it's created during build time by parsing `%REACT_APP_NAME%-use-cache-buster`, so another value on `REACT_APP_NAME` will break this completely.